hive/build.gradle

```diff
@@ -0,0 +1,26 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '6.0.0'
+    id 'java'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+  compile "org.apache.hive:hive-jdbc:2.1.0:standalone"
```

Comment:
We need to use Hive 2.3.7 and 3.1.2 versions. Maybe we need to have 2 modules - one for each hive version if they are binary incompatible.

Reply:
Filed an issue as a reminder for us to upgrade Hive to 2.3.7, using 2.1.0 for now.